src/explore-education-statistics-admin/src/types/ckeditor.ts

```diff
@@ -30,6 +30,9 @@ export interface EditorConfig {
   heading?: {
     options: HeadingOption[];
   };
+  link?: {
+    decorators: { [key: string]: LinkDecoratorAutomatic | LinkDecoratorManual };
```

Comment:
Not a big deal, but with all of these index types, you can refactor them to use `Dictionary` which is a little more succinct:

```ts
decorators: Dictionary<LinkDecoratorAutomatic | LinkDecoratorManual>;
attributes: Dictionary<string>;
styles?: Dictionary<string>;
```